src/update-feeds/update-transactions.ts

```diff
@@ -141,21 +141,16 @@ export const estimateMulticallGasLimit = async (
   return ethers.BigNumber.from(2_000_000);
 };
 
-export const deriveSponsorWallet = (sponsorWalletMnemonic: string, dapiName: string) => {
+export const getDerivedSponsorWallet = (sponsorWalletMnemonic: string, dapiName: string) => {
```

Comment:
Had to split this function because it is stateful (checks the cache for private keys), but we want to use the `deriveSponsorWallet` as an utility in e2e test contract setup.